### PR TITLE
{lang}[system/system] Perl v5.38.0

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Depends on:
- https://github.com/easybuilders/easybuild-easyblocks/pull/3010

### CODE REVIEW

File: easybuild/easyconfigs/p/Perl/Perl-5.38.0.eb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13'),
+]
+
+# need to take care of $CFLAGS ourselves with SYSTEM toolchain
```
[the remark this answers is not in the document]
Comment (SebastianAchilles) [reply easybuild/easyconfigs/p/Perl/Perl-5.38.0.eb:25]: I think in the `SYSTEM` toolchain `$CFLAGS` is not set.

We have to do the same also for `ncurses` and `zlib` in the `SYSTEM` toolchain:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/z/zlib/zlib-1.2.13.eb#L21
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/ncurses/ncurses-6.4.eb#L28

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jscclxc1.int.jsc-clx.fz-juelich.de - Linux Rocky Linux 9.3, x86_64, Intel Xeon Processor (Cascadelake) (cascadelake), Python 3.9.18
See https://gist.github.com/SebastianAchilles/1895c9abbc1a8364530f5b2d406cde76 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/9208479e52a6409cd236cd437a8b6ab3 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**